worker/echo/main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/google/uuid"
```

Comment:
This import line is in the wrong place. Running `goimports` on the package moves
this line down, outside the list of stdlib imports.

Reply:
Fixed.